src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/AutomatedPanningMachine.java

```diff
@@ -61,15 +61,16 @@ public AutomatedPanningMachine(ItemGroup itemGroup, SlimefunItemStack item) {
     @Override
     public void onInteract(Player p, Block b) {
         ItemStack input = p.getInventory().getItemInMainHand();
+        boolean isGoldPanInput = goldPan.isCorrectInputMaterial(input.getType());
 
-        if (SlimefunUtils.isItemSimilar(input, new ItemStack(Material.GRAVEL), true, false) || SlimefunUtils.isItemSimilar(input, new ItemStack(Material.SOUL_SAND), true, false)) {
+        if (isGoldPanInput || netherGoldPan.isCorrectInputMaterial(input.getType())) {
```

Comment:
Semantically speaking a `GoldPanType` enum might not be a bad idea

Reply:
I didn't use an enum, but I feel like it is cleaned up nicely as is
If you disagree I suppose I could refactor it once more